README.md

```diff
@@ -9,6 +9,12 @@ drush pm:enable os2forms_organisation
 
 Edit settings on `/admin/os2forms_organisation/settings`.
 
+**NOTE** This module creates and uses a user field called `field_organisation_user_id`.
+It is up to the using party to ensure it is filled out with the SF1500
+organisation  `bruger` ID. If it is not, nothing is displayed.
```

Comment:
Trim space after “organisation”. Your text editor should have a “Fill paragraph” command (like https://www.emacswiki.org/emacs/FillParagraph, say) that can help you clean this up.

“bruger” should probably be quoted with regular quotes, i.e. “bruger”, and not quoted as inline code.

Can we provide a link to more details on “the SF1500 organisation  `bruger` ID”?

Reply:
Updated.